src/test/java/edu/ucla/library/iiif/fester/fit/PostCsvFIT.java

```diff
@@ -356,6 +363,72 @@ public final void testDeleteUploadedFilesOnEnd(final TestContext aContext) throw
             });
         }
 
+        /**
+         * Tests the alpha-numeric sorting of manifests on a collection.
+         *
+         * @param aContext A test context
+         */
+        @Test
+        public final void testCollectionManifestWorkSorting(final TestContext aContext) {
+            final Async asyncTask = aContext.async();
+            final String collectionS3Key = IDUtils.getCollectionS3Key("ark:/21198/zz0025hqmb");
+
+            // PUT a collection manifest (that already has work manifests on it) in S3
+            myS3Client.putObject(BUCKET, collectionS3Key, PROTESTA_COLLECTION_MANIFEST);
+
+            // POST a work CSV with randomly sorted rows; all work manifests generated from these rows should be ordered
+            // before all those already existing on the collection manifest
+            postCSV(WORKS_CSV_PROTESTA_1, post -> {
+                if (post.succeeded()) {
+                    final HttpResponse<Buffer> postResponse = post.result();
+                    final int postStatusCode = postResponse.statusCode();
+                    final String postStatusMessage = postResponse.statusMessage();
+
+                    if (postStatusCode == HTTP.CREATED) {
+                        final String getPath = IDUtils.getResourceURIPath(collectionS3Key);
+
+                        // Retrieve the updated manifest and check it
+                        myWebClient.get(FESTER_PORT, Constants.UNSPECIFIED_HOST, getPath).send(get -> {
+                            if (get.succeeded()) {
+                                final HttpResponse<Buffer> getResponse = get.result();
+                                final int getStatusCode = getResponse.statusCode();
+                                final String getStatusMessage = getResponse.statusMessage();
+
+                                if (getStatusCode == HTTP.OK) {
+                                    final JsonObject json = get.result().bodyAsJsonObject();
+                                    final List<Collection.Manifest> works = Collection.fromJSON(json).getManifests();
+                                    final ManifestLabelComparator comparator = new ManifestLabelComparator();
+
+                                    for (int i = 0; i < works.size() - 1; i++) {
+                                        aContext.assertTrue(comparator.compare(works.get(i), works.get(i + 1)) < 0);
+                                    }
+
+                                    if (!asyncTask.isCompleted()) {
+                                        asyncTask.complete();
+                                    }
+                                } else {
+                                    aContext.fail(
```

Comment:
Wondering about shortening collectionS3Key to just `s3Key` since we're getting pretty far over to the right side (and wrapping). The method is dealing with collections so think `s3Key` would be sufficient.